services/src/main/java/org/keycloak/services/clientpolicy/executor/SecureRedirectUriEnforceExecutor.java

```diff
@@ -34,26 +33,34 @@
 import org.keycloak.services.clientpolicy.context.DynamicClientRegisterContext;
 import org.keycloak.services.clientpolicy.context.DynamicClientUpdateContext;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
 public class SecureRedirectUriEnforceExecutor implements ClientPolicyExecutorProvider {
 
     private static final Logger logger = Logger.getLogger(SecureRedirectUriEnforceExecutor.class);
+    private static final String LOGMSG_PREFIX = "CLIENT-POLICY";
```

Comment:
LOGMSG_PREFIX constant is defined individually on almost every class (SecureRedirectUriEnforceExecutor, SecureSessionEnforceExecutor etc). If it should be used by multiple classes, then maybe will be better to just have a constant for it?

Another more generic point is, that there is some flexibility how can be logging configured in the standalone.xml in logging subsystem. You can configure logging format for various classes to use the thread name, method name etc. I am not 100% sure if ClientPolicyLogger is not duplicating things, which can be handled/configured by the logging framework itself. But I am fine with the current approach in this PR. Maybe logging is something to revisit in the future...

Reply:
@mposolda As for `LOGMSG_PREFIX`, I will find out where it is used and replace static fields with one constant defined in some appropriate class.

As for this PR's custom logging configuration (add class name, method name, etc...), if the underlying logging framework can do that, it might appropriate for this framework to do that. However, as you mentioned, this task will be treated in the future.

Reply:
Yes, I suggest to possibly improve logging in the future if needed. That was just some concern for the future...